exercises/perfect-numbers/example.jl

```diff
@@ -1,6 +1,10 @@
 function aliquot_sum(n)
     if !isinteger(n) || n < 0
         throw(DomainError(n,"n must be a non-negative integer"))
+    # In Julia <1.4, you need this special case because `sum` doesn't like
```

Comment:
A workaround would be `reduce(+, i for i in 1:div(n, 2) if n % i == 0; init=0)` but it's slightly slower.

Reply:
I want to keep the workaround separate to emphasise that there is a nice option for modern Julias.